helpers/resetPasswordTamplate.js

```diff
@@ -0,0 +1,55 @@
+import dotenv from "dotenv";
+dotenv.config();
+
+const resetPasswordTamplate = token => {
+  const tamplate = `
+    <html>
+        <head>
+            <title></title>
+            <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+            <meta name="viewport" content="width=device-width, initial-scale=1" />
+            <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        </head>
+
+        <body style="background-color: #eee; padding:30px 0">
+            <div
+            style=" margin: 50px auto; width: 400px; background-color: white;padding: 20px;color: #f4f4f4; border-radius: 5px;box-shadow: 0 1px 3px rgba(0,0,0,0.12), 0 1px 2px rgba(0,0,0,0.10); "
+            >
+            <div border="0" cellpadding="0" cellspacing="0" width="100%">
+                <h2
+                style="color:rgb(0,47,255); text-align: center; font-size: 20px; font-family: Helvetica, Arial, sans-serif;"
+                >
+                Titan Devs
```

Comment:
Author's Heaven, because it's the application that is sending the email not the team. @abayo-luc

Reply:
Got it! Thanks.